apps/meteor/client/views/room/modals/FileUploadModal/FileUploadModal.tsx

```diff
@@ -60,6 +62,11 @@ const FileUploadModal = ({
 			});
 		}
 
+		if (description.length >= maxCharactersAllowed) {
```

Comment:
Shouldn't it be `>` only? Shouldn't we accept it if it's equals to the limitation?